server/controllers/apply/applicationsController.test.ts

```diff
@@ -245,78 +242,193 @@ describe('applicationsController', () => {
       expect(response.redirect).toHaveBeenCalledWith(paths.applications.searchByPrisonNumber({}))
```

Comment:
I'm a bit confused by this - are we really expecting a redirect to the search by prison number page when a create call is successful?

Reply:
Good spot, this is a mistake, will fix.